forui/lib/src/widgets/badge/badge_content.dart

```diff
@@ -0,0 +1,63 @@
+part of 'badge.dart';
+
+@internal final class FBadgeContent extends StatelessWidget {
+
+  final FBadgeStyle style;
+  final String label;
+
+  const FBadgeContent(this.label, {required this.style, super.key});
```

Comment:
Should we make `label` named? I think it's more within flutter's general convention.